### PR TITLE
Item names

### PR DESCRIPTION
**Pull request by weisspe**

### CODE REVIEW

File: ui/00_message/common/item_name.xml [hunk 1]
```diff
@@ -4101,63 +4101,63 @@
...
   </message>
   <message>
     <key>ITEM_NAME_1029</key>
-    <text xml:space="preserve">ルナサーガ</text>
+    <text xml:space="preserve">Lunar Saga</text>
   </message>
   <message>
     <key>ITEM_NAME_1030</key>
-    <text xml:space="preserve">ヴァリアントホーン</text>
+    <text xml:space="preserve">Variant Horn</text>
```
Comment (BananaFlambe) [inline ui/00_message/common/item_name.xml]: Are you sure it's Variant and not Valiant?

File: ui/00_message/common/item_name.xml [hunk 2]
```diff
@@ -201,35 +201,35 @@
...
+    <text xml:space="preserve">Ghost Amulet</text>
   </message>
   <message>
     <key>ITEM_NAME_54</key>
-    <text xml:space="preserve">戦神のタリスマン</text>
+    <text xml:space="preserve">Demon Chaos Talisman</text>
   </message>
   <message>
     <key>ITEM_NAME_55</key>
-    <text xml:space="preserve">守護神のタリスマン</text>
```
Comment (BananaFlambe) [inline ui/00_message/common/item_name.xml:224]: Why change it to amulet if it says talisman?

EDIT: I saw there is another guardian deity amulet with the "amulet" katakana so it makes sense to write talisman here

**Comment by BananaFlambe** [comment]
Please check the comments I've left, after you fix those it's ready to be merged,I've checked everything    and looks good to me. Good work!

**Comment by weisspe** [comment]
Good catch on those!  I bet it is valiant instead.  Also I did the charm/amulet/talismans in a batch and must have jumbled up that one.

**Comment by Crevox** [comment]
Please be sure to be checking Dark Arisen for potential localizations that already exist. For example, they use the term "Periapt" for multiple items, but none of yours used it. The item names could be different this time around, but, just something to watch for.

**Comment by weisspe** [comment]
I'll update these tomorrow to use periapt. Is there a good resource to
check for dark arisen? I've always been a PC guy so I'm aware of the game
but haven't been able to play/experience it to know some of the things that
aren't obvious in the raw translation.

**Comment by Crevox** [comment]
Best one I can provide right now is the wikis.

EN: http://dragonsdogma.wikia.com/wiki/Dragon%27s_Dogma_Wiki
JP: http://www10.atwiki.jp/dragons_dogma/pages/1.html

Riftcrystal also ripped the EN/JP versions of the game I believe using the arc tool available online, and sometimes references those.

Talisman is not necessarily wrong because there are some items called Talisman in Dark Arisen, but the different item types do different things. :)

**Pull request merged**